ansible/roles/druid-ingestion/defaults/main.yml

```diff
@@ -35,3 +35,7 @@ sl_observation_evidence_task_count: 1
 sl_survey_task_count: 1
 rollup_course_completion_audit_syncts_taskcount: 1
 rollup_ml_project_taskcount: 1
+ml-observation-status-rollup_taskcount: 1
```

Comment:
We don't use `-` in these variable names also can you follow the pattern similar to existing variables. 
Rename this **ml-observation-status-rollup_taskcount** to **rollup_ml_observation_status_taskcount**
Similary update all variables.

Reply:
Modified as suggested